sdk/appconfiguration/app-configuration/samples/readme.md

```diff
@@ -0,0 +1,144 @@
+# App Configuration client library for TypeScript/JavaScript
```

Comment:
I like your main README. I think for your samples README, you can focus more on how someone can run your samples in their own project.
I won't claim this is the _best_ example but you can see how we've done this in event hubs:
https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/samples

Reply:
I've updated it quite a bit. Take a look and let me know if that's what you were thinking of.

Reply:
I agree

Reply:
@ramya-rao-a  - take a look at the updates - does that seem better?